git/fetchpack.go

```diff
@@ -26,7 +26,7 @@ type FetchPackOptions struct {
 	Thin                           bool
 	IncludeTag                     bool
 	UploadPack                     string
-	Depth                          int32
+	Depth                          int
```

Comment:
Why is this changed from int32 to int? The officially documentation implies that it's a 32 bit integer:

```
Limit fetching to ancestor-chains not longer than n. git-upload-pack treats the special depth 2147483647 as infinite even if there is an ancestor-chain that long
```

Reply:
I did this so that we could pass a pointer to the option to flags.IntVar(). Otherwise, we need a temp variable and then a conversion. I'll put this back the the way it was.